Networking/Networking/Model/Product/ProductBundleItem.swift

```diff
@@ -71,4 +71,17 @@ public enum ProductBundleItemStockStatus: String, Codable, GeneratedFakeable {
     case inStock        = "in_stock"
     case outOfStock     = "out_of_stock"
     case onBackOrder    = "on_backorder"
+
+    /// Returns the localized text version of the Enum
+    ///
+    public var description: String {
+        switch self {
+        case .inStock:
+            return NSLocalizedString("In stock", comment: "Display label for the bundle item's inventory stock status")
+        case .outOfStock:
+            return NSLocalizedString("Out of stock", comment: "Display label for the bundle item's inventory stock status")
+        case .onBackOrder:
+            return NSLocalizedString("On back order", comment: "Display label for the bundle item's inventory stock status")
+        }
+    }
```

Comment:
Not a super big deal, but probably this localization would be better a better fit at the UI layer, not at the networking one 😁

Reply:
Good call, I moved that to the UI layer in 5862ffe.